python/pyproject.toml

```
@@ -43,3 +79,53 @@ exclude = '''
     dist
 )/
 '''
+
+[tool.isort]
+line_length = 79
+multi_line_output = 3
+include_trailing_comma = true
+force_grid_wrap = 0
+combine_as_imports = true
+order_by_type = true
+known_dask = [
+    "dask",
+    "distributed",
+    "dask_cuda",
+]
+known_rapids = [
+    "nvtext",
+    "cudf",
+    "cuml",
+    "cugraph",
+    "dask_cudf",
+]
+known_first_party = [
+    "rmm",
+]
+default_section = "THIRDPARTY"
+sections = [
+    "FUTURE",
+    "STDLIB",
+    "THIRDPARTY",
+    "DASK",
+    "RAPIDS",
+    "FIRSTPARTY",
+    "LOCALFOLDER",
+]
+skip = [
+    "thirdparty",
+    ".eggs",
+    ".git",
+    ".hg",
+    ".mypy_cache",
+    ".tox",
+    ".venv",
+    "_build",
+    "buck-out",
+    "build",
+    "dist",
+    "__init__.py",
+]
+
+[tool.setuptools]
```

Using the `license-files` option from [these config options]( https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html#setuptools-specific-configuration ) should pick up the license file

```suggestion
[tool.setuptools]
license-files = ["LICENSE"]
```

---

Just a note for other reviewers, the license file will automatically be picked up by the default license files setting but we are going to set this anyway to be explicit and protect against future changes since this feature is still in beta in setuptools.